src/Articles.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "./utils/api";
+
+function Articles() {
+  const [articles, setArticles] = useState([]);
+  useEffect(() => {
+    getArticles().then((articleData) => {
+      setArticles(articleData);
+    });
+  });
```

Comment:
This is the source of your multiple console logs - you must have a dependency array - empty if you only want to run the effect once